src/PKSim.Core/Services/DistributedParameterToTableParameterConverter.cs

```diff
@@ -275,20 +290,61 @@ private bool parameterShouldBeDefinedAsTable(IDistributedParameter parameter)
          return !parameter.NameIsOneOf(CoreConstants.Parameters.MEAN_HEIGHT, CoreConstants.Parameters.MEAN_WEIGHT);
       }
 
-      private void createPopulationTableParameter(KeyValuePair<string, IDistributedParameter> individualParameter, PopulationSimulation populationSimulation,
-         IEnumerable<ParameterDistributionMetaData> allDistributionsForMaleParameter, IEnumerable<ParameterDistributionMetaData> allDistributionsForFemaleParameter)
+      private void createPopulationTableParameter(
+         string individualParameterPath,
+         IDistributedParameter individualParameter,
+         PopulationSimulation populationSimulation,
+         IReadOnlyList<ParameterDistributionMetaData> allDistributionsForMaleParameter,
+         IReadOnlyList<ParameterDistributionMetaData> allDistributionsForFemaleParameter)
       {
-         if (populationSimulation == null) return;
-         var parameter = individualParameter.Value;
+         if (populationSimulation == null)
+            return;
 
-         addAgingDataToPopulationSimulation(populationSimulation, individualParameter.Key, parameter,
+         addAgingDataToPopulationSimulation(populationSimulation, individualParameterPath, individualParameter,
             p =>
             {
-               var distributions = allDistributionsWithAgeStrictBiggerThanOriginData(allDistributionsForMaleParameter, allDistributionsForFemaleParameter, p.OriginData).ToList();
+               var distributions = allDistributionsWithAgeStrictBiggerThanOriginData(allDistributionsForMaleParameter, allDistributionsForFemaleParameter, p.OriginData);
                return createTableFormulaFrom(p, distributions);
             });
       }
 
+      private void createPopulationHeightTableParameter(
+         string heightParameterPath,
+         IDistributedParameter meanHeightParameter,
+         PopulationSimulation populationSimulation
+      )
+      {
+         if (populationSimulation == null || !_createAgingDataInSimulation)
+            return;
```

Comment:
As far as I understand `_createAgingDataInSimulation `is responsible for creation of Table parameters in **population** simulations. 
I would rename it then to something like `_createAgingDataInPopulationSimulation`. And also rename the corresponding parameter in all calling functions:
* here in `UpdateBuildConfigurationForAging`
* in `BuildConfigurationTask.CreateFor`

Otherwise it's highly confusing.

Reply:
apart from that: looks good for me :)

Reply:
I had renamed the variable in the interface already
![image](https://user-images.githubusercontent.com/1041237/75040553-dbe93280-5488-11ea-9095-65ca921e3ae5.png)

But not th elocal one. I'll do it and then merge